upgrade/steps-helpers.go

```diff
@@ -121,6 +121,10 @@ func prBody(ctx Context, repo ProviderRepo,
 		fmt.Fprintf(b, "- Upgrading pulumi-terraform-bridge/pf from %s to %s.\n",
 			goMod.Pf.Version, targetBridge)
 	}
+
+	if ctx.UpgradeSdkVersion {
+		fmt.Fprintf(b, "- Upgrading pulumi SDK to `latest`.\n")
```

Comment:
These two code snippets are not related - `ctx.UpgradeSdkVersion` is by default not true when we are upgrading the bridge.
In fact, we may see conflicting PRs in case we pass `--kind=sdk` in addition to `---kind=all` (and due to bridge dependencies, using the former is probably generally ill-advised to begin with.)

I did consider adding this here about but I decided it wasn't worth it:

a) we never use `--kind=sdk` in our automation
b) if we do run it locally, the version will be easily seen in the diff, and it is is always going to be `latest` because it's a simple `go get`
c) if we _do_ add a field to the `GoMod` struct that is `.Sdk` now we're in the business of tracking and updating that parallel to the steps, because in the case of bridge upgrading, this field changes depending on when you read it. Not impossible, just a lot of extra for something that isn't a routine part of our automation.